INFO_ON_MERGE_CONFLICTS.md

```diff
@@ -0,0 +1,163 @@
+# Info on merge conflicts <!-- omit in toc -->
```

Comment:
What do you think about just calling the file `MERGE_CONFLICTS.MD`?

Reply:
Done – thanks!